pkgs/top-level/all-packages.nix

```
@@ -11586,7 +11586,8 @@ in
   inherit (beam.interpreters)
```

```suggestion
  recurseIntoAttrs beam.interpreters;
```

I think this should be enough already. See https://github.com/NixOS/nixpkgs/blob/7248f89da2dab8bca3a9f4f11d0407d0c8e25e89/pkgs/top-level/all-packages.nix#L11578